OpenRA.Mods.Common/Traits/CashTrickler.cs

```diff
@@ -56,6 +58,7 @@ public CashTrickler(CashTricklerInfo info)
 		protected override void Created(Actor self)
 		{
 			resources = self.Owner.PlayerActor.Trait<PlayerResources>();
+			showTicks = info.ShowTicks && self.OccupiesSpace != null;
```

Comment:
Thinking about it, the better approach here would probably be to add a lint error instead of failing silently.

Reply:
:+1:

This should be doable in an `IRulesetLoaded` method, rather than a specific custom lint test.

Reply:
Yeah, I was referring to that.